frontends/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "ipa-frontend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "next -p 7759c",
+    "build": "next build",
+    "build:css": "postcss src/style/tailwind.css -o src/style/main.css",
+    "build:watch": "postcss src/tailwind.css -o src/main.css --watch",
+    "lint": "npm run eslint -- js,jsx,json",
+    "format": "npm run format:js && npm run format:css",
+    "format:js": "npm run lint -- --fix",
+    "start": "next start"
```

Comment:
the instructions in the README don't work. I'm getting the following error when running `npm start`:

```
Error: Could not find a production build in the '/Users/cfelder/carforyou/IPA-Projekt/frontends/.next' directory. Try building your app with 'next build' before starting the production server
````

Maybe this is why you committed the `.next` folder initially. It is generated by a `npm run build` and used for production deployments. For a local run, you can just resort to `npm run dev`, also see https://nextjs.org/docs/getting-started#manual-setup

Please update the README so the instructions for starting the drone works